### PR TITLE
Greet Users on First Created PR or Issue

### PR DESCRIPTION
**Pull request by Xtigyro**
Automatically greet users on first created PR or issue by them.

### CODE REVIEW

**Comment by Xtigyro** [comment]
CC: @underscorgan @Iristyle @slconley @scottcressi @mwaggett @nwolfe @adrienthebo @dhollinger

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Message that will be displayed on users'' first issue'
+        pr-message: 'Message that will be displayed on users'' first pr'
```
Comment (underscorgan) [inline .github/workflows/greetings.yml:13]: We should update these messages to what we actually want them to say :) Would be good to find some other examples maybe

Comment (Xtigyro) [reply .github/workflows/greetings.yml:13]: Gotcha - will do some more research. And yeah - totally forgot to change those, ha-ha, thanks!

Comment (mwaggett) [reply .github/workflows/greetings.yml:13]: I wonder if we already have some community standards for greetings - @binford2k ?

Comment (binford2k) [reply .github/workflows/greetings.yml:13]: We don't yet. There are some projects using https://probot.github.io/apps/welcome/ right now.

Comment (Xtigyro) [reply .github/workflows/greetings.yml:13]: Ladies and gentlemen, should I try to come up with something or it's better an epic in Jira to be opened and start off from there?

**Pull request closed without merging**